build-tools/check-boot-times/Program.cs

```diff
@@ -0,0 +1,593 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.CommandLineUtils;
```

Comment:
I would suggest using `Mono.Options` instead. :-)

Multiple apps within this repo already use it for command-line parsing, e.g.:

https://github.com/xamarin/xamarin-android/blob/master/build-tools/api-merge/api-merge.cs#L21-L58

https://github.com/xamarin/xamarin-android/blob/e74670375fd90b016851e83040f00a75ff36021b/tools/jit-times/Program.cs#L27-L61

[Docs](http://docs.go-mono.com/?link=T%3aMono.Options.OptionSet).

Reply:
Thanks for the suggestion, as we discussed offline, I will stick Microsoft.Extensions.CommandLineUtils since it is the way Microsoft DotNet team is suggesting and also all DotNet CLI tools are using it.